src/js/views/responses/item.js

```diff
@@ -126,6 +126,11 @@ function($, _, Backbone, events, settings, util, api, Responses, template) {
         } else if (question.type === 'text') {
```

Comment:
This came in with the change for editing text fields (2ccf87a8c0a47fa8cf8d994dca8f14952490d1b6). Was the intention to allow adding notes if the original response had no notes? If so, we may need to allow empty text fields here and prevent display in the template (which has different modes for read-only display mode and edit-mode display, as i recall).